_layouts/default.html

```diff
@@ -170,10 +170,15 @@ <h1 class="text-light">Send word our way</h1>
                   <i class="fas fa-calendar-alt"></i>
                 </a>
               </li>
+              <li class="list-inline-item">
+                <a href="https://www.youtube.com/channel/UCILxIdh3uyw64JjtdRmdtVg" target="_blank" rel="noopener" aria-label="Code Connector's Youtube">
```

Comment:
nitpick: YouTube is actually the correct capitalization

Reply:
Got me, thanks this was a great learning experiance for me!